### PR TITLE
Created basic type/model generation for net package

### PR DESCRIPTION
**Pull request by RachelTucker**

### CODE REVIEW

File: ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/model/response/BaseResponse.java [hunk 1]
```diff
@@ -21,15 +21,13 @@
 
 public class BaseResponse {
 
-    private final NetHelper netHelper;
+    private final NetHelper netHelper = NetHelper.getInstance();
```
Comment (rpmoore) [inline ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/model/response/BaseResponse.java]: Why was this changed?

Comment (RachelTucker) [inline ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/model/response/BaseResponse.java]: I didn't see the need for passing in the net helper. I can change it back if you prefer

Comment (rpmoore) [inline ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/model/response/BaseResponse.java]: The main reason is for testability.  By injecting the implementation we can mock the implementation in a test and and then pass it in.  Unless this is something that we are not doing (though we probably should be...).
